modules/sandboxed-containers-installing-operator-cli.adoc

```diff
@@ -55,55 +85,30 @@ spec:
 +
 [NOTE]
 ====
-Using the *preview-1.1* channel ensures that you install the version of {sandboxed-containers-first} that is compatible with your {product-title} version.
+Using the *preview-{sandboxed-containers-version}* channel ensures that you install the version of {sandboxed-containers-first} that is compatible with your {product-title} version.
```

Comment:
This gets rendered as 1.1, will need to be 1.2 for GA

Reply:
See above comment.